gum/backend-libdwarf/gumsymbolutil-libdwarf.c

```diff
@@ -93,6 +93,8 @@ struct _GumDieDetails
   Dwarf_Debug dbg;
 };
 
+static gboolean gum_find_nearest_symbol_by_address (const gpointer address,
```

Comment:
```suggestion
static gboolean gum_find_nearest_symbol_by_address (gconstpointer address,
```

But seeing as we're using `gpointer` in the surrounding code it's probably better to be consistently doing it wrong until we fix it in the other places :)